srcalc/src/main/webapp/js/displayResults.js

```diff
@@ -6,6 +6,7 @@ function initESigDialog(){
     var electronicSignatureDialog = $('.eSigDialog').dialog({
     	beforeClose: function(event, ui) {
     		$('#eSigInput').val('');
```

Comment:
I think a comment would be helpful for the future. E.g., "Reset the controls for the next use of the dialog."